### ISSUE TITLE
Unsupported CPU Error On M2 Mac

### ISSUE THREAD

**Issue by rat-moonshine** [DominoVagrant/demo#9]
**Hardware: M2 Mac
OS: macOS Monterey**

While tried to install VBox with installer titled "macOS / Intel hosts" from https://www.virtualbox.org/wiki/Downloads, it throws me following error:
![image](https://user-images.githubusercontent.com/24370870/197688581-366aeef4-3dda-4e27-8b45-aaca13d9409b.png)

Thus, I installed by the installer titled "Developer preview for macOS / Arm64 (M1/M2) hosts" from VBox website and this time it installed without an error.

However, when running `vagrant up`, at some point process terminates with following error:

```
==> domino-vm: Running 'pre-boot' VM customizations...
==> domino-vm: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "8edf6de0-db1a-45a1-a39c-5d3e508a1fbf", "--type", "headless"]

Stderr: VBoxManage: error: Unsupported CPU. (VERR_UNSUPPORTED_CPU)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

**Comment by piotrzarzycki21** [comment]
There is Developer preview of VirtualBox 7.0 for M1 -> https://www.virtualbox.org/wiki/Downloads

**Comment by JoelProminic** [comment]
Yeah, it looks like @rat-moonshine tried this already ("Developer preview for macOS / Arm64 (M1/M2) hosts").   It sounds like this build is specifically supposed to fix this problem, though, so I wonder if there was some other configuration issue.

@JesseProminic, please test the ["Developer preview for macOS / Arm64 (M1/M2) hosts"](https://www.virtualbox.org/wiki/Downloads) build on your local macOS (not the HostMyApple) to see if this works for you.   If it works, it should make your testing easier.

**Comment by rat-moonshine** [comment]
"Developer preview for macOS / Arm64 (M1/M2) hosts" -- as it words, it's a developer preview and not the stable one. If you opens the VBox.app separately, you can see a warning on possible instability, as well.
